frontend/src/hooks/useReceivingCaptions.tsx

```diff
@@ -0,0 +1,48 @@
+import { Subscriber } from '@vonage/client-sdk-video';
+import { useState, useEffect, useMemo } from 'react';
+
+export type CaptionsType = {
+  caption: string;
+  isReceivingCaptions: boolean;
```

Comment:
What's the distinction between `caption` and `isReceivingCaptions`?  Given it's not intuitively obvious, should we add some comments here?